Update bobores.cfg

MrSnooker opened the pull request.

MrSnooker committed:
Update bobores.cfg

Firerazer reviewed (changes requested):
Hello,
The string like "__ENTITY__gold-ore__" etc, more generaly all string having __ in their structure must not be translated.
Please review your translation

Karosieben committed:
Update bobores.cfg

Do not translate the "__ENTITY__..." values. It's a reference to the other values.
